examples/example_plot_spectra_accessors.py

```diff
@@ -0,0 +1,58 @@
+"""
+Plotting wave spectra data, using the accessor syntax
+================================================
```

Comment:
The documentation is going to fail here, for some reason the `===` need to be equal or longer than the title.